packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/delete-connector/delete-connector.controller.js

```diff
@@ -0,0 +1,45 @@
+export default class DeleteConnectorCtrl {
+  /* @ngInject */
+  constructor($translate, DatabaseService) {
+    this.DatabaseService = DatabaseService;
+    this.$translate = $translate;
+  }
+
+  $onInit() {
+    this.trackDashboard('connector::actions_menu::delete_connector', 'page');
+  }
+
+  cancel() {
+    this.trackDashboard('connector::actions_menu::delete_connector_cancel');
```

Comment:
**question**: Is the tracking label correct?
Maybe it's `connector::actions_menu::delete_connector**::**cancel`

Reply:
I think it is `delete_connector_cancel`, to match what has been done before: https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/storages/databases/database/indexes/delete-index/delete-index.controller.js#L15
However, your suggestion makes sense to me, so I'll ask for a check on tracking once all features are merged on the main branch. Then if some tracking strings need to be changed, I'll create a PR to handle this.
Does it sound good to you ?

Reply:
Yes, it would be good to handle the tracking changes together.